drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -276,6 +292,45 @@ def get_catalog(ra, dec, sr=0.1, catalog='GSC241'):
     return r_csv
 
 
+def get_catalog_from_footprint(footprint, catalog='GSC241'):
+    """ Extract catalog from VO web service based on the specified footprint
+
+    Parameters
+    ----------
+    footprint : numpy.ndarray
+        Array of RA, Dec points that describe the footprint polygon
+
+    catalog : str, optional
+        Name of catalog to query, as defined by web-service.  Default: 'GSC241'
+
+    Returns
+    -------
+    csv : CSV object
+        CSV object of returned sources with all columns as provided by catalog
+
+    """
+    serviceType = 'vo/CatalogSearch.aspx'
+    spec_str = 'STCS=polygon{}&FORMAT={}&CAT={}&MINDET=5'
+    headers = {'Content-Type': 'text/csv'}
+    fmt = 'CSV'
+
+    footprint_string = ""
+    for item in footprint:
+        footprint_string += "%20{}%20{}".format(item[0],item[1])
+    spec = spec_str.format(footprint_string, fmt, catalog)
+    serviceUrl = '{}/{}?{}'.format(SERVICELOCATION, serviceType, spec)
+    rawcat = requests.get(serviceUrl, headers=headers)
```

Comment:
Since this is a VO service, I assume it is public (i.e., any user could run this without an account)?

Reply:
Correct. It is a publically accessible service. It doesn't require any sort of special credentials or previously set up account.